src/linux.rs

```diff
@@ -127,6 +144,14 @@ impl LinuxI2CDevice {
         Ok(())
     }
 
+    /// Set the slave address for this device, even if it is already in use
+    /// by a driver!
+    pub fn force_set_slave_address(&mut self, slave_address: u16) -> Result<(), LinuxI2CError> {
```

Comment:
Why would you need this method?

Reply:
In case the new address is already occupied, but we still need to switch to it?

Reply:
The question would be why would you want to switch the device you are talking to.
One would rather create a new `I2CDevice` with the other address, forcefully or not.

I understand the motivation in the original issue to forcefully create an `I2CDevice` and talk to a device that is independently already blocked. However, this method would be the opposite side of the fan, using one `I2CDevice` to actually talk to several devices, which is why I do not understand the motivation.

Reply:
Makes sense- I'll make it private.

Reply:
Could you also mark it as `unsafe`?